vllm/model_executor/models/config.py

```diff
@@ -489,6 +489,14 @@ def verify_and_update_config(cls, vllm_config: "VllmConfig") -> None:
             logger.info("Using bfloat16 kv-cache for DeepSeekV3.2")
 
 
+class Qwen3MoeForCausalLMConfig(VerifyAndUpdateConfig):
+    @staticmethod
+    def verify_and_update_config(vllm_config: "VllmConfig") -> None:
+        structured_outputs_config = vllm_config.structured_outputs_config
+        if structured_outputs_config.reasoning_parser == "":
```

Comment:
How do you start vLLM?

This field structured_outputs_config.reasoning_parser can be set via the parameter --reasoning-parser qwen3.